src/com/company/DefaultHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.company;
```

Comment:
We need some unit tests on this class. Let's discuss when we pair!

Reply:
We spoke about this yesterday. It is much better to test at a handler level than test a full hit to the running server for every test of handler behavior. 

The server itself should have a test for each endpoint to see that it is handled. And the handlers themselves should have a test to verify each behavior of the handler itself